spk/homeassistant/Makefile

```diff
@@ -8,7 +8,6 @@ SPK_DEPENDS = "python310"
 BUILD_DEPENDS = cross/python310
 
 DEPENDS += cross/gevent
-DEPENDS += cross/homeassistant.pillow
```

Comment:
homeassistant still depends on cross/pillow

Reply:
But it is, the proposal here is to have it under `python310` which makes it available not only for `homeassistan` but as well for `beets` which is noarch.

When testing, if `python310` provides it, it must be at version 8.2.0 as otherwise `homeassistant` tentative to downgrade it into its local crossenv fails, and thus the application overall doesn't work out.

Reply:
BTW, I noticed that `octoprint` as a different method to install wheels at post-install which allows it to download + install a local copy independently from the version provided by default by the python package.  Just a thought if that could prove useful.

Otherwise, as is, the basic `homeassistant` testing I did looked promising.